Datadog/Datadog.xcodeproj/project.pbxproj

```diff
@@ -6855,6 +6859,7 @@
 				ASSETCATALOG_COMPILER_APPICON_NAME = AppIcon;
 				CLANG_ENABLE_MODULES = YES;
 				CODE_SIGN_STYLE = Automatic;
+				DEVELOPMENT_TEAM = JKFCB4CN7C;
```

Comment:
There are 3 diffs on `DEVELOPMENT_TEAM` that need to be reverted (so it defaults to automatic team selection).

Reply:
oh good catch, thanks!

Reply:
Also, to test on device `DEVELOPMENT_TEAM` is needed 👍, but better way to pass this setting is through `Datadog.local.xcconfig` (gitignored):
```
DEVELOPMENT_TEAM[sdk=iphoneos*]=JKFCB4CN7C
```

Reply:
`⌘C`/`⌘V` 😄  Thank you!!